### ISSUE TITLE
Example invalid

### ISSUE THREAD

**Issue by SharePointBlackBelt** [MicrosoftDocs/OfficeDocs-SharePoint-PowerShell#171]
What does the example in this article have to do with Get-SPEnterpriseSearchMetadataCrawledProperty?
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 60645b5d-fac6-4f02-1cd8-c76ceda53cdb
* Version Independent ID: acea67c9-6ea9-a577-510b-694f7d2e6f87
* Content: [Get-SPEnterpriseSearchMetadataCrawledProperty (sharepoint-server)](https://docs.microsoft.com/en-us/powershell/module/sharepoint-server/get-spenterprisesearchmetadatacrawledproperty?view=sharepoint-server-ps)
* Content Source: [sharepoint/sharepoint-server-ps/sharepoint-server/Get-SPEnterpriseSearchMetadataCrawledProperty.md](https://github.com/MicrosoftDocs/OfficeDocs-Sharepoint-PowerShell/blob/main/sharepoint/sharepoint-server-ps/sharepoint-server/Get-SPEnterpriseSearchMetadataCrawledProperty.md)
* Product: **sharepoint-server-itpro**
* GitHub Login: @Techwriter40
* Microsoft Alias: **pamgreen**

**Comment by get-itips** [comment]
Hello @SharePointBlackBelt how are you?

Just to confirm, do you refer to this example?

`$ssa = Get-SPEnterpriseSearchServiceApplication`

`$cat = Get-SPEnterpriseSearchMetadataCategory -SearchApplication $ssa -Identity People`

`Get-SPEnterpriseSearchMetadataCrawledProperty -SearchApplication $ssa -Category $cat -Limit 1`

I tested it and it does what the description says, please kindly let me know if I understood you correctly,

Thanks

**Comment by SharePointBlackBelt** [comment]
Yes, you are right. I should have phrased this better... the problem is that it does not show what the crawled property mapping is, and the article doesn't mention how to get the mapping.
For example, running something like:
$ssa = Get-SPEnterpriseSearchServiceApplication
Get-SPEnterpriseSearchMetadataCrawledProperty -SearchApplication $ssa -Category $cat -Name ows_Proposal_x0020_Section
Returns:
Name               : ows_Proposal_x0020_Section
CategoryName       : SharePoint
Propset            : 00130329-0000-0130-c000-000000131346
IsMappedToContents : True
VariantType        : 0
showing it's mapped, but mapped to what? Maybe the article could be improved with additional examples.

**Comment by get-itips** [comment]
Hello @SharePointBlackBelt Vincent,

> the problem is that it does not show what the crawled property mapping is, and the article doesn't mention how to get the mapping

I understand what you were looking for now, thanks.

Is this method, of the resulting CrawledProperty object, the mapping you are looking for?

https://docs.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ms562672(v=office.14)

**Comment by SharePointBlackBelt** [comment]
Oh yeah, that works. Thank you. Maybe this example, or one similar could be added to the article.
$crawledprop = Get-SPEnterpriseSearchMetadataCrawledProperty -SearchApplication $ssa -Category $cat -Name ows_Proposal_x0020_Section
$crawledprop.GetMappedManagedProperties()
Name                 PID      ManagedType  EnabledForScoping  EqualityMatchOnly
----                 ---      -----------  -----------------  -----------------
ProposalSection      1012     Text         False              False

**Comment by get-itips** [comment]
@SharePointBlackBelt 
Glad to hear Vincent.
Yes will propose to add an example in line with the first example.
Thank you
